internal/generate/packagemanifest/packagemanifest.go

```diff
@@ -173,7 +172,7 @@ func (g Generator) makeBaseGetter(inputDir string) func() (*apimanifests.Package
 
 // makePkgManFileName will return the file name of a PackageManifest.
 func makePkgManFileName(operatorName string) string {
-	return strings.ToLower(operatorName) + packageManifestFileExt
+	return operatorName + packageManifestFileExt
```

Comment:
Do we no longer need to lowercase this?

Reply:
We should assume `operatorName` is lowercased at this point.